guvnor-m2repo-editor/guvnor-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/GuvnorM2RepositoryTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2015 JBoss Inc
```

Comment:
It's a cherry-pick - I wasn't going to go through and modify the dates... unless you insist!